shared/src/main/java/gov/hhs/cdc/trustedintermediary/wrappers/HapiFhir.java

```diff
@@ -13,4 +15,8 @@ <T extends IBaseResource> T parseResource(String fhirResource, Class<T> clazz)
             throws FhirParseException;
 
     String encodeResourceToJson(Object resource);
+
+    List<Base> evaluate(IBaseResource root, String expression);
```

Comment:
This method, and the implementation in `HapiFhirImplementation` is only used in tests.  This method should be deleted and any tests that specifically test this method can be deleted.

Reply:
Before removing, I wanted to make sure this is not something that we will be needing in the near future

Reply:
YAGNI - done

Reply:
Heck yeah!  💥